package.json

```diff
@@ -31,13 +34,13 @@
     "postcss-loader": "^3.0.0",
     "remove-files-webpack-plugin": "^1.1.3",
     "sass-loader": "^7.3.1",
+    "style-loader": "^2.0.0",
```

Comment:
This is actually not needed as we are using custom styles for Photoswipe, I'll remove it.